cmd/influxd/upgrade/testdata/v1db/wal/mydb/autogen/8/.gitignore

```diff
@@ -0,0 +1,4 @@
+# Ignore everything in this directory
```

Comment:
Please remove all of these `.gitignore` files. Perhaps we can help you achieve what you were intending to do with these file a different way?

Reply:
These files were added to allow committing the wal folder with empty dirs. As stated below, this will be cleaned.